src/robot/rebot.py

```diff
@@ -201,6 +201,11 @@
                           match (starting from 1).
                           Examples: --tagstatlink mytag:http://my.domain:Title
                           --tagstatlink "bug-*:http://url/id=%1:Issue Tracker"
+    --autoexpandkeywords name:<pattern>|tag:<pattern> *
+                          Matching keywords will be initially expanded in log.html.
+                          For pattern matching rules see --removekeywords option.
+                          Example: --autoexpandkeywords name:CapturePageScreenshot.
```

Comment:
The `.` at the ends should probably be removed.

Reply:
fixed